evm/src/generation/partial_trie.rs

```diff
@@ -22,6 +23,7 @@ pub enum PartialTrie {
     Leaf { nibbles: Nibbles, value: Vec<u8> },
 }
 
+#[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
```

Comment:
Hm it's not obvious to me why an ordering would be useful or which one would be most natural, is it just for `BTreeMap`?

Reply:
Yeah `PartialOrd` and `Ord` aren't needed downstream. I was just thinking since this is part of the public interface, someone might want to order these for whatever reason. I'm fine taking the ord traits out if it's not really clear what the default impl should be.